### PR TITLE
adjusting frontend track and adding proper category names to each group

### PR DESCRIPTION
**Pull request by RazShutyWMDE**

### CODE REVIEW

File: trackData/frontend.json [hunk 1]
```diff
@@ -61,6 +61,8 @@
         ],
         "examples": [
             "Implemented a way to steer away from our Jquery/OOUI and enable us to work with a more modern framework",
+            "Architected the new SSR with Vue.js on the wikidata team",
+            "Architected a new frontend using a new of-the-shelf framework",
```
Comment (AlaaSarhan) [inline trackData/frontend.json]:
```suggestion
            "Architected a new frontend using a new off-the-shelf framework",
```

**Review by AlaaSarhan — APPROVED** [review]

File: trackData/frontend.json [hunk 2]
```diff
@@ -61,6 +61,8 @@
         ],
         "examples": [
             "Implemented a way to steer away from our Jquery/OOUI and enable us to work with a more modern framework",
+            "Architected the new SSR with Vue.js on the wikidata team",
```
Comment (jakobw) [inline trackData/frontend.json]: My first reaction was that this sounds oddly specific compared to the other examples. If the idea is to look at those and go "oh, I've done that so I fall into this category!" then this example has very limited reach. It's also quite backend-y for a frontend example depending on how you want to look at it.

```suggestion
            "Understands the challenges of server-side rendering for modern frameworks like Vue or React, and is able develop sustainable solutions for them",
```
^ is my attempt at a generalization of the example. "sustainable" is very fluffy, but I somehow wanted to convey that there are multiple solutions  - thinking of how we tried the php renderer, no ssr, trying to reduce template duplication, ... - and that the trick is to find good trade-offs, as usual :)

File: trackData/frontend.json [hunk 3]
```diff
@@ -74,7 +76,6 @@
         ],
         "examples": [
             "Defined and drove migration strategy to a new framework",
-            "Architected the new SSR with Vue.js on the wikidata team",
             "Understands all new frontend frameworks and chose the right one for the right task"
```
Comment (tzhelyazkova) [inline trackData/frontend.json]:
```suggestion
             "Understands the differences between frontend frameworks and their ecosystems and choses the right one for the right task"
```
What I mean by "ecosystems" is all the surrounding tools and plugins that frameworks come with, and also the community around the framework. That might not be the best wording.

File: trackData/frontend.json [hunk 4]
```diff
@@ -61,6 +61,8 @@
         ],
         "examples": [
             "Implemented a way to steer away from our Jquery/OOUI and enable us to work with a more modern framework",
+            "Architected the new SSR with Vue.js on the wikidata team",
+            "Architected a new frontend using a new of-the-shelf framework",
             "Built reusable components that are used all over the ogranisation"
```
Comment (tzhelyazkova) [inline trackData/frontend.json]:
```suggestion
             "Built reusable components that are used all over the organization",
             "Incorporated the usage of Vuex, Nuxt, Storybook and other tools into the development process"
```

File: trackData/frontend.json [hunk 5]
```diff
@@ -61,6 +61,8 @@
         ],
```
Comment (tzhelyazkova) [inline trackData/frontend.json:61]: GitHub is being weird and doesn't let me add a comment/suggestion in the upper row.
I think we should add one more line in `"signals"`, something like `"Researches and suggests new tools for building better front end solutions by reducing the complexity and amount of code being written"`

Comment (tzhelyazkova) [reply trackData/frontend.json:61]: Also `"Has a good understanding of javascript bundlers and how to use them"`

Comment (RazShutyWMDE) [reply trackData/frontend.json:61]: +1

**Pull request merged**